src/internal/packager/git/gitea.go

```diff
@@ -196,6 +214,31 @@ func (g *Git) CreatePackageRegistryToken() (CreateTokenResponse, error) {
 	return createTokenResponse, nil
 }
 
+func (g *Git) UpdateGiteaPVC() (string, error) {
+	create := "true"
```

Comment:
wasn't sure i wanted to make this var, but wanted the return value to make more sense. Maybe a better name for the method would accomplish that.